src/vs/workbench/contrib/mergeEditor/browser/view/mergeEditor.ts

```diff
@@ -567,14 +545,13 @@ export class MergeEditorResolverContribution extends Disposable {
 		this._register(editorResolverService.registerEditor(
 			`*`,
 			{
-				id: MergeEditorInput.ID,
-				label: localize('editor.mergeEditor.label', "Merge Editor"),
+				id: DEFAULT_EDITOR_ASSOCIATION.id,
```

Comment:
Nice. We probably want the changes from https://github.com/microsoft/vscode/pull/155696/files also incorporated in this PR, specifically to remove the override and hacks I had in place.

Reply:
I pushed that.